src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -800,7 +800,7 @@ public void submoduleClean(boolean recursive) throws GitException, InterruptedEx
     	if (recursive) {
             args.add("--recursive");
     	}
-    	args.add("git clean -fdx");
+        args.add("git clean -ffdx");
```

Comment:
I'd rather not have the indentation change.  Can you put the white space on the front of the line back the way it was before?